core/pom.xml

```diff
@@ -136,6 +136,11 @@
       <groupId>org.codehaus.plexus</groupId>
       <artifactId>plexus-interpolation</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
```

Comment:
why is this needed ? Actually we are trying to move away from org.json (we did in dmp already) as there are licensing issues with this library. We should at least avoid to add it, instead add the gson library as it is currently only included via a transitive dependency on d-m-p. This dependency should be made explicit.